localisation/00_zuwc_united_localisation.csv

```diff
@@ -3016,11 +3012,21 @@ EVTOPTB_WCADD_1100;I'll pay [target_crystal_owner.GetHerHim] a visit!;;Ich werde
+DYN11_DESC;With a score of $SCORE$ you$FAILED$ beat House Stormpike. This dwarven clan's origin is not well known, though it is safe to assume that they appeared some time after the War of Three Hammers as subjects of the ruling Bronzebeard clan. They are a well respected clan and are known for their fine and discerning tastes and riches. Under Vanndar Stormpike's leadership, they would lead an expedition into Alterac valley to defend it from the savage Frostwolves while also keeping an eye out for relics of their past and their profitable mining endeavours.;;Mit einem Punktestand von $SCORE$ ist es Euch$FAILED$ gelungen, das Haus Stormpike zu �bertreffen. �ber den Ursprung dieses Zwergenklans ist wenig bekannt, obwohl man davon ausgehen kann, dass sie einige Zeit nach dem Krieg der drei H�mmer als Untertanen des herrschenden Bronzebeard-Klans auftraten. Sie sind ein angesehener Klan und f�r ihren feinen und anspruchsvollen Geschmack und Reichtum bekannt. Unter der F�hrung von Vanndar Stormpike sollten sie eine Expedition ins Alteractal f�hren, um es vor den wilden Frostw�lfen zu verteidigen und gleichzeitig ein waches Auge auf die Relikte ihrer Vergangenheit und ihre gewinnbringenden Bergbaubem�hungen zu haben.;;;;;;;;;;;x
+DYN12_DESC;With a score of $SCORE$ you$FAILED$ beat house Mechagon. In the distant past, this family founded the monarchy of Gnomeregan. The last king of the gnomes, King Mechagon, however, abandoned his people with a small entourage in search of a mystical land of forgotten technology, which they later settled and named after him. Under his rule in Mechagon, his people have managed to remove a large problem of their society, an aging population, through partially turning themselves into mechanical versions of themselves.;;Mit einem Punktestand von $SCORE$ ist es Euch$FAILED$ gelungen, das Haus Mechagon zu �bertreffen. In der fernen Vergangenheit gr�ndete diese Familie die Monarchie von Gnomeregan. Der letzte K�nig der Gnome, K�nig Mechagon, verlie� jedoch sein Volk mit einem kleinen Gefolge auf der Suche nach einem mystischen Land vergessener Technologie, das sie sp�ter besiedelten und nach ihm benannten. Unter seiner Herrschaft ist es seinem Volk in Mechagon gelungen, ein gro�es Problem ihrer Gesellschaft, eine alternde Bev�lkerung, zu �berwinden, indem sie sich selbst zum Teil in mechanische Versionen verwandelten.;;;;;;;;;;;x
```

Comment:
Standardizing capitalized House in `beat House xxx`
```suggestion
DYN12_DESC;With a score of $SCORE$ you$FAILED$ beat House Mechagon. In the distant past, this family founded the monarchy of Gnomeregan. The last king of the gnomes, King Mechagon, however, abandoned his people with a small entourage in search of a mystical land of forgotten technology, which they later settled and named after him. Under his rule in Mechagon, his people have managed to remove a large problem of their society, an aging population, through partially turning themselves into mechanical versions of themselves.;;Mit einem Punktestand von $SCORE$ ist es Euch$FAILED$ gelungen, das Haus Mechagon zu �bertreffen. In der fernen Vergangenheit gr�ndete diese Familie die Monarchie von Gnomeregan. Der letzte K�nig der Gnome, K�nig Mechagon, verlie� jedoch sein Volk mit einem kleinen Gefolge auf der Suche nach einem mystischen Land vergessener Technologie, das sie sp�ter besiedelten und nach ihm benannten. Unter seiner Herrschaft ist es seinem Volk in Mechagon gelungen, ein gro�es Problem ihrer Gesellschaft, eine alternde Bev�lkerung, zu �berwinden, indem sie sich selbst zum Teil in mechanische Versionen verwandelten.;;;;;;;;;;;x
```

Reply:
and thx one last time :D